nbs/examples/MLFlow_and_neuralforecast.ipynb

```diff
@@ -0,0 +1,466 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    model_uri = f"runs:/{run.info.run_id}/model"
</pre><p>This is different from the model_uri above (the one above is <code>neuralforecast-model</code>)</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Nixtla/neuralforecast/pull/892/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='11'/>

Reply:
That is correct, the first uri call creates three folders under .../mlruns:

1. The one is .../mlruns/0 , here a new ID register is created every time a the model is run (ideally with different hyper parameters). under this folder a new run is created, this creates a path like this: C:/Users/Dev/Documents/mlruns/0/409bdaa8ccf44caab4c74fc4ed63a392/artifacts/model, where the last part of the path is by default model.
2. There is another folder created, .../mlruns/model, this folder holds general information about the model model being trained and get the name from the model_uri created during the run, and on the case of this example, adds the folder with the same name as the end of the model_uri (neuralforecast-model).
3. The third folder is, .../mlruns/trash, if I recall correctly.
Thus, to load the model using MLFlow, you can either use model_uri = f"runs:/{run.info.run_id}/model" which points to the last run ID, or call the run ID that you would like to load as per the instructions on below (refer also to screenshot attached):

 logged_model = 'runs:/409bdaa8ccf44caab4c74fc4ed63a392/model'
 loaded_model = mlflow.pyfunc.load_model(logged_model)

I added a clarification via commit https://github.com/Nixtla/neuralforecast/pull/892/commits/eb866dfa88995726fdd5c5b975811f27d916db02

![MLFLow_model_load](https://github.com/Nixtla/neuralforecast/assets/138342606/d2c4b1a0-47a7-4230-804e-79fc17e0c802)